site/content/3.10/data-science/llm-knowledge-graphs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Large Language Models (LLMs) and Knowledge Graphs
+menuTitle: Large Language Models and Knowledge Graphs
+weight: 133
+description: >-
+  Integrate large language models (LLMs) with knowledge graphs using ArangoDB
+archetype: default
+---
+{{< description >}}
+
+Large language models (LLMs) and knowledge graphs are two prominent and
+contrasting concepts and each possesses unique characteristics and functionalities
```

Comment:
curious if you think this sounds better (otherwise feel free to ignore):

```suggestion
contrasting concepts, each possessing unique characteristics and functionalities
```